src/components/AppContentListItem/AppContentListItem.vue

```diff
@@ -0,0 +1,57 @@
+<!--
+ - @copyright Copyright (c) 2019 Christoph Wurst <christoph@winzerhof-wurst.at>
+ -
+ - @author Christoph Wurst <christoph@winzerhof-wurst.at>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<template>
+	<router-link class="app-content-list-item" :to="to">
+		<div v-if="$slots.icon"
+			class="app-content-list-item-icon">
```

Comment:
I would actually prefer the use of a prop for that. :wink:
If we want to support Avatar, then a slot for avatar only and add checks to make sure the component in the avatar slot have the name `avatar` :grin: 
Same goes for the texts, this is supposed to be text, so let's not add the option to have anything else. This goes for
- line-one
- line-two
- details

Actions make sense to have a slot for. Though could be default slot. :thinking:

Reply:
Mail needs a line-two slot as it's adding icons for replied messages and ones with attachments. A prop would not allow that.

Reply:
I'll use a custom component for Mail